tsconfig.json

```diff
@@ -3,10 +3,13 @@
         "outDir": "build",
         "module": "esNext",
         "target": "es6",
-        "allowJs": true,
-        "noImplicitAny": false,
+        "allowJs": false,
+        "noImplicitAny": true,
+        "noImplicitReturns": true,
         "jsx": "preserve",
         "moduleResolution": "node",
+        "allowSyntheticDefaultImports": true,
```

Comment:
To accommodate the react types. Without it:
```
TS1192: Module '"/Users/Shared/workspace/seatmaps-client/node_modules/@types/react/index"' has no default export.
```

Reply:
Ah, I see. Because this changes how modules are resolved, let's make sure we analyze the bundle for unused imports. I could see this complicating tree shaking quite a bit.